zkevm-circuits/src/copy_circuit.rs

```diff
@@ -92,6 +104,12 @@ pub struct CopyCircuitConfig<F> {
     /// Since `src_addr` and `src_addr_end` are u64, 8 bytes are sufficient for
     /// the Lt chip.
     pub addr_lt_addr_end: LtConfig<F, 8>,
+    /// Lt chip to check: src_addr < src_addr_end.
+    /// Since `src_addr` and `src_addr_end` are u64, 8 bytes are sufficient for
+    /// the Lt chip.
+    pub is_word_index_end: LtConfig<F, 1>,
```

Comment:
This variable is used with the meaning of "continue" or "not last", so the name is confusing.

```suggestion
    pub is_word_continue: LtConfig<F, 1>,
```

Reply:
good name, i like it!

Reply:
updated in [ed4d3e4](https://github.com/scroll-tech/zkevm-circuits/pull/466/commits/ed4d3e42c56768d42a576083de5615deaaf85705) thanks!